src/azure-cli/azure/cli/command_modules/iot/tests/latest/test_iot_commands.py

```diff
@@ -709,6 +709,24 @@ def test_hub_file_upload(self, resource_group, resource_group_location, storage_
                  .format(hub, rg, containerName, storageConnectionString),
                  expect_failure=True)
 
+        # Testing hub update without $default storage endpoint
```

Comment:
Add a test changing the authentication type to identity

Reply:
Good point, I have a test to prove a non-fileupload update works without erroring, but since we've marked it as a fix for a bug that's appeared more than once, might as well add that test (or at least a specific test using the `--set property` function).